source/rules/literals.rs

```diff
@@ -201,9 +201,9 @@ pub enum StringError {
 
 named!(
     pub string<Literal>,
-    alt_complete!(
-        call!(string_single_quoted)
-      | call!(string_nowdoc)
+    alt!(
```

Comment:
I know this should not be part of this PR but it's a single commit, sorry for that.